README.md

```diff
@@ -289,6 +297,7 @@ Visiting `http://localhost:4567/` should show a Yoti Connect button
 * Activity Details
   * [X] User ID `user_id`
   * [X] Base64 Selfie URI `base64_selfie_uri`
+  * [X] Age verified `age_verified`
```

Comment:
This is just retaining the old method of retrieving this, right? So we don't need to mention getting it by the attribute name

Reply:
We are mentioning getting the original attribute by name, in case they want to get the sources and verifiers.

Reply:
I see now 👍